src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -196,9 +180,9 @@ void OwncloudAdvancedSetupPage::updateStatus()
         if (dirNotEmpty) {
             t += tr("<p><small><strong>Warning:</strong> The local folder is not empty. "
```

Comment:
Isn't this text kind of redundant to the new widget?

I think as a user, I wouldn't be sure to know whether both are related. Also, I would have to ask myself, "what is this resolution they're talking about?", it doesn't really get clear from the text what it wants me, the user, to do.

Reply:
This is the default behaviour if you select a non empty dir, the message box only shows up if you try to use a forbidden dir.
But yes the messing popping up below the choices is 🤣